frontend/components/Course/InstructorQueuePage/CreateQueue/CreateQueue.tsx

```diff
@@ -126,7 +125,6 @@ const CreateQueue = (props: CreateQueueProps) => {
             backFunc();
             successFunc();
         } catch (e) {
-            logException(e);
```

Comment:
Currently createQueue is implemented to throw an error for unexpected responses, and is currently getting caught to update an error state variable. I can change it to instead return a status promise similar to what I did with createQuestion to eliminate the try catch In general this is a case of refactoring the return to be consistently the status so that we don;t catch the error.

Reply:
Ah you're right, my bad, mixed up createQueue with createQuestion. Let's leave it like this